### ISSUE TITLE
测试集准确率

### ISSUE THREAD

**Issue by jxy-001** [manideep2510/siamese-BERT-fake-news#4]
训练能达到90 ，但测试总是60左右，您知道可能是什么原因么？

**Comment by manideep2510** [comment]
For me the training accuracy never touched 90%, it went up till 81%. And just a small note, I have trained on both Train and Validation set combined and tested on the Test set.
